PowerEditor/installer/nsisInclude/uninstall.nsh

```diff
@@ -54,7 +54,7 @@ Function un.onUninstSuccess
 	RMDir "$INSTDIR\"
 
 	RMDir "$APPDATA\${APPNAME}\plugins\"
-	RMDir "$installPath\autoCompletion\"
+	RMDir "$INSTDIR\autoCompletion\"
```

Comment:
Should this line be before `RMDir "$INSTDIR\"` ?

Reply:
@donho Yes, you are right. I could not catch it because in my testing NppShell_06.dll was locked. So Notepad++ directory was not deleted. I should have not assumed only NppShell_06.dll as reason.
I have fixed it now.

However, It is good to remove NppShell_06.dll as well on installed. So that Notepad++ dir also will be removed.